pennylane/drawer/mpldrawer.py

```diff
@@ -255,15 +257,15 @@ class MPLDrawer:
     _cwire_scaling = 0.25
     """The distance between successive control wires."""
 
-    def __init__(self, n_layers, n_wires, c_wires=0, wire_options=None, figsize=None, fig=None):
+    def __init__(self, n_layers, wires, c_wires=0, wire_options=None, figsize=None, fig=None):
```

Comment:
We could also make this `wire_order` and not accept passing an integer representing `n_wires` anymore, but this allows more things that worked before to keep working, like `MPLDrawer(1, 2)`. 

If there were no established precedent, I would do that, because I think it's more clear. But I wasn't sure how much we want to deviate from what it already accepts.

Reply:
Actually we need some info from `wire_map` in `tape_mpl`, so we'll need to use that instead. Never mind.